### PR TITLE
feat: add manager classes handling consents logic

### PR DESCRIPTION
**Pull request by cuadradek**
feat(core): add manager classes handling consents logic

- New manager classes handling consents logic were added to every layer.
They contain no methods so far.

feat(openapi): add new manager handling consents logic

- New manager handling consents logic were added, it contains no
methods so far.

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java [hunk 1]
```diff
@@ -256,4 +256,10 @@
 	 * @return ConfigManagerBl
 	 */
 	ConfigManagerBl getConfigManagerBl();
+
+	/**
+	 * Gets a ConsentsManagerBl
+	 * @return ConsentsManagerBl
+	 */
+	ConsentsManagerBl getConsentsManagerBl();
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java:264]: I think getConsentsManager() method should be here too, same as other managers have it.

File: perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java [hunk 2]
```diff
@@ -115,7 +118,7 @@
 	private SearcherBl searcherBl = null;
 	private TasksManagerBl tasksManagerBl = null;
 	private ConfigManagerBl configManagerBl = null;
-
+	private ConsentsManagerBl consentsManagerBl = null;
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java:121]: should be probably added to toString() method also

Comment (cuadradek) [reply perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java:121]: Done, I added missing ConfigManager as well.

File: perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java (hunk 1 above)
Comment (cuadradek) [reply perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java:278]: Done, I added missing getConfigManager() as well.

**Review by xflord — APPROVED** [review]

**Pull request merged**